src/services/util/helpers.tsx

```diff
@@ -41,3 +41,6 @@ export function formatNumberToBigint(
 ) {
   return BigInt(BigNumber(amount).shiftedBy(precision).toString());
 }
+
+export const catTextToLen = (text: string, len: number) =>
```

Comment:
🐱 

catTextToLen -> cutTextToLength
len -> length

I'd suggest to avoid shorting any words at all in the future.